cockroachdb_dedicated/assets/dashboards/cockroach_cloud_serverless_overview.json

```diff
@@ -0,0 +1,2552 @@
+        "x": 0,
+        "y": 27,
+        "width": 12,
+        "height": 22
+      }
+    },
+    {
+      "id": 678470534448650,
+      "definition": {
+        "title": "Changefeeds",
+        "background_color": "blue",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 5203120089548500,
+            "definition": {
+              "title": "Changefeed Status",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "alias": "Running",
+                      "formula": "default_zero(query1)"
+                    },
+                    {
+                      "alias": "Failures",
+                      "formula": "default_zero(query2)"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "query": "sum:crdb_cloud.changefeed.running{$Cluster}.as_count()",
+                      "data_source": "metrics",
+                      "name": "query1"
+                    },
+                    {
+                      "query": "sum:crdb_cloud.changefeed.failures{$Cluster}.as_count()",
+                      "data_source": "metrics",
+                      "name": "query2"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ]
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 4,
+              "height": 3
+            }
+          },
+          {
+            "id": 6766355742123040,
+            "definition": {
+              "type": "note",
+              "content": "The number of all changefeeds by status - Running, Paused, and Failed. The currently running changefeeeds includes sinkless.\n\nMonitor and alert on the Paused changefeeds metric to safeguard against an inadvertent operational error of leaving a changefeed job in a paused state for an extended period of time. Changefeed jobs should not be paused for a long time because the protected timestamp prevents garbage collection.\n\nThe Failed changefeeds metrics tracks the permanent changefeed job failures that the jobs system will not try to restart. Any increase in this counter should be investigated. An alert on this metric is recommended.",
```

Comment:
```suggestion
              "content": "The number of all changefeeds by status - Running, Paused, and Failed. The currently running changefeeds includes sinkless.\n\nMonitor and alert on the Paused changefeeds metric to safeguard against an inadvertent operational error of leaving a changefeed job in a paused state for an extended period of time. Changefeed jobs should not be paused for a long time because the protected timestamp prevents garbage collection.\n\nThe Failed changefeeds metrics tracks the permanent changefeed job failures that the jobs system will not try to restart. Any increase in this counter should be investigated. An alert on this metric is recommended.",
```